PostFreeSurfer/scripts/FreeSurfer2CaretConvertAndRegisterNonlinear.sh

```diff
@@ -213,6 +213,12 @@ for GrayordinatesResolution in ${GrayordinatesResolutions} ; do
     ${CARET7DIR}/wb_command -volume-label-import "$AtlasSpaceFolder"/ROIs/wmparc."$GrayordinatesResolution".nii.gz ${SubcorticalGrayLabels} "$AtlasSpaceFolder"/ROIs/ROIs."$GrayordinatesResolution".nii.gz -discard-others
     [ "${T2wImage}" != "NONE" ] && applywarp --interp=spline -i "$AtlasSpaceFolder"/"$AtlasSpaceT2wImage".nii.gz -r "$AtlasSpaceFolder"/ROIs/Atlas_ROIs."$GrayordinatesResolution".nii.gz -o "$AtlasSpaceFolder"/"$AtlasSpaceT2wImage"."$GrayordinatesResolution".nii.gz
     applywarp --interp=spline -i "$AtlasSpaceFolder"/"$AtlasSpaceT1wImage".nii.gz -r "$AtlasSpaceFolder"/ROIs/Atlas_ROIs."$GrayordinatesResolution".nii.gz -o "$AtlasSpaceFolder"/"$AtlasSpaceT1wImage"."$GrayordinatesResolution".nii.gz
+    applywarp --interp=nn -i "$AtlasSpaceFolder"/brainmask_fs.nii.gz -r "$AtlasSpaceFolder"/ROIs/Atlas_ROIs."$GrayordinatesResolution".nii.gz -o "$AtlasSpaceFolder"/brainmask_fs."$GrayordinatesResolution".nii.gz
+    ${CARET7DIR}/wb_command "((Brainmask-1)*-1)*CIFTIStandardSpace" "$AtlasSpaceFolder"/ROIs/MissingGrayordinates.nii.gz -var Brainmask "$AtlasSpaceFolder"/brainmask_fs."$GrayordinatesResolution".nii.gz -var CIFTIStandardSpace "$AtlasSpaceFolder"/ROIs/Atlas_ROIs."$GrayordinatesResolution".nii.gz
+    MissingGrayordinates=`fslstats "$AtlasSpaceFolder"/ROIs/MissingGrayordinates.nii.gz -V | awk '{print $1}'`
+    echo "MissingGrayordinates" > "$AtlasSpaceFolder"/ROIs/MissingGrayordinates.txt
+    echo "${MissingGrayordinates}" >> "$AtlasSpaceFolder"/ROIs/MissingGrayordinates.txt
```

Comment:
Could MissingGrayordinates.txt be generated in a way that additional includes lines that quantify the number of missing grayordinates per CIFTI subcortical ROI?  That would be very valuable for quickly distinguishing between missing brainstem vs. missing other structures.

Reply:
I don't know an easy way to do it, but am happy if someone else wants to do it.

Reply:
To get all structures, convert the brain mask to the cifti voxel space, import to cifti, parcellate with the subcortical labels and SUM, -cifti-convert -to-text.  Could generate the list of names starting with -*-label-export-table.  Stitching them together could get interesting.

If we specifically expect brainstem to be an issue, we could just hardcode its label value into the script to get stats on it alone (or everything except it).

Reply:
I would prefer the latter solution.